### PR TITLE
Add optional --out-dir parameter to the report script

### PR DESCRIPTION
**Pull request by fvaleri**
This is useful if you don't have enough /tmp space and you want to use a different dir. If you don't specify the out dir, then the current behavior is maintained.

### CODE REVIEW

File: tools/report.sh [hunk 1]
```diff
@@ -8,9 +8,13 @@ fi
 { # this ensures the entire script is downloaded #
 NAMESPACE=""
 CLUSTER=""
+BRIDGE=""
+CONNECT=""
+MM2=""
 KUBECTL_INSTALLED=false
 OC_INSTALLED=false
 KUBE_CLIENT="kubectl"
+OUT_DIR="$(mktemp -d)"
```
Comment (scholzj) [inline tools/report.sh:17]: Should this be called only when `--out-dir` was not specified?

Comment (fvaleri) [reply tools/report.sh:17]: This is a default value, which is overwritten when you specify `--out-dir`.

Comment (scholzj) [reply tools/report.sh:17]: Right. But you call `mktemp -d` then even if you end up not needing it.

Comment (fvaleri) [reply tools/report.sh:17]: Agree. We want to avoid creating folders when they are not used. Let me fix that.

**Review by scholzj — APPROVED** [review]
Thanks.

**Review by ppatierno — APPROVED** [review]

**Pull request merged**